lib/classes/ssl/class.lescript.php

```diff
@@ -517,6 +528,13 @@ public function getLastLinks()
 		preg_match_all('~Link: <(.+)>;rel="up"~', $this->lastHeader, $matches);
 		return $matches[1];
 	}
+
+	public function getAgreementURL()
+	{
+		preg_match_all('~Link: <(.+)>;rel="terms-of-service"~', $this->lastHeader, $matches);
+		return $matches[1][0];
```

Comment:
You are absolutely right... Solution would be to add `if (!empty($this->license))` before sending the reg update. I wanted to change this before @d00p merged, but too late now :) Maybe on for a new PR.